### DISCUSSION TITLE
Stuck on loading page

### DISCUSSION THREAD

**Discussion by bobokun** [jesec/flood#236] [Q&A]
Hi I've been trying to get flood working and I'm using a container that combines qbittorrent and flood UI.
When Loading the flood UI I am stuck on the loading page but I do see the bars moving. Even after leaving it on it does not end up loading. 

![image](https://user-images.githubusercontent.com/12660469/110541322-94663080-80f5-11eb-8507-254dfed55e81.png)

**Comment by jesec** [comment]
Refresh.

Check logs of Flood server. And, F12->Network tab and check requests and responses.

Generally, this is caused by a misconfiguration of client connection settings. You probably have to check your configurations or contact the maintainer of that Docker container.

**Reply by bobokun** [reply]
When I click on the network tab I don't see anything that stands out...
![image](https://user-images.githubusercontent.com/12660469/110637252-311ee180-817b-11eb-8619-986dc5e44d70.png)
However when I go to preview I do see a message that I need to enable JavaScript to run this app
![image](https://user-images.githubusercontent.com/12660469/110637364-527fcd80-817b-11eb-8cbd-cb5b54e92691.png)

Response:
`<!doctype html><html lang="en"><head><meta charset="utf-8"/><meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate"/><meta http-equiv="Pragma" content="no-cache"/><meta http-equiv="Expires" content="0"/><meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no"/><meta name="theme-color" content="#000000"/><meta name="apple-mobile-web-app-title" content="Flood"/><meta name="apple-mobile-web-app-capable" content="yes"/><meta name="apple-mobile-web-app-status-bar-style" content="default"/><link rel="manifest" href="manifest.json"/><link rel="shortcut icon" href="favicon.ico"/><link rel="apple-touch-icon" sizes="180x180" href="icon_maskable_180x180.png"/><title>Flood</title><script defer="defer" src="static/js/main.ace3a2ad.js"></script><link href="static/css/main.f0e7dbd2700465a64ed3.css" rel="stylesheet"></head><body><noscript>You need to enable JavaScript to run this app.</noscript><div id="app"></div></body></html>`

**Reply by jesec** [reply]
Preview inside network tab is not relevant. 

Refresh and you should see something like this:

![image](https://user-images.githubusercontent.com/11585141/110637850-d2904c80-81e8-11eb-9443-c33cb0164d29.png)

If there is nothing out of the ordinary, Flood frontend is properly loaded. You can then click the `activity-stream` and inspect the EventStream. If there is no event at all, Flood server is not properly connected to your torrent client.

You should check the logs of Flood server as well. Again, it is highly likely that there are misconfigurations at some point. It is recommended to double check the documentation of your Docker image, filesystem privileges/contexts and the client connection settings.

**Reply by bobokun** [reply]
This is what I see, it looks like it is something is going on in the activity stream, I see the size slowly growing
 but that's the only activity I see.
![image](https://user-images.githubusercontent.com/12660469/110641433-01bea380-8180-11eb-9030-c59a3e0f4cb0.png)

![image](https://user-images.githubusercontent.com/12660469/110641676-477b6c00-8180-11eb-827b-edb10e80b1f0.png)


Unfortunately I'm having trouble locating the logs for Flood in the container I am using. I'm using the qFlood container managed by hotio

**Reply by jesec** [reply]
OK. It seems that Flood server is not responding with the torrent list. You would have to check the client connection settings. You probably misconfigured it somehow. Additionally, make sure that you got the filesystem privileges/contexts right. You should also try to raise the issue to the maintainer of your container image, as they may be more familiar with the easy-to-go-wrong points of their container images.

**Reply by bobokun** [reply]
Thanks, will do.
